tests/spread/plugins/craft-parts/matter-sdk/task.yaml

```
@@ -0,0 +1,51 @@
+summary: Craft Parts matter SDK plugin test
+manual: true
+kill-timeout: 180m
+
+systems:
+  - ubuntu-22.04-64
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "$SNAP/snap/snapcraft.yaml"
+
+restore: |
+  cd "$SNAP"
+  snapcraft clean
+  rm -f ./*.snap
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+execute: |
+  cd "$SNAP"
+
+  # Build and install the snap
+  snapcraft
+  snap install "${SNAP}"*.snap --dangerous
```

shellcheck should be OK with this:

```suggestion
  snap install "${SNAP}*.snap" --dangerous
```